test-jest/cartesian/Area.spec.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { FC } from 'react';
+import { render } from '@testing-library/react';
+import { Area, Surface } from '../../src';
+
+describe('<Area />', () => {
+  const data: any = [
+    { x: 10, y: 50, value: 100 },
+    { x: 50, y: 50, value: 100 },
+    { x: 90, y: 50, value: 100 },
+    { x: 130, y: 50, value: 100 },
+    { x: 170, y: 50, value: 100 },
+  ];
+
+  test('Render customized label when label is set to be a function', () => {
+    const renderLabel = (props: { index: number; x: number; y: number }) => {
+      const { index, x, y } = props;
+
+      return (
+        <text x={x} y={y} className="customized-label">
+          {index}
+        </text>
+      );
+    };
+    const { container } = render(
+      <Surface width={500} height={500}>
+        <Area dataKey="value" isAnimationActive={false} baseLine={200} points={data} label={renderLabel} />
+      </Surface>,
+    );
+
+    expect(container.querySelectorAll('.customized-label')).toHaveLength(data.length);
+  });
+
+  test('Render customized label when label is set to be a react element', () => {
+    const Label: FC<{ index: number; x: number; y: number }> = props => {
+      const { index, x, y } = props;
+
+      return (
+        <text x={x} y={y} className="customized-label">
+          {index}
+        </text>
+      );
+    };
+    const { container } = render(
+      <Surface width={500} height={500}>
+        <Area
+          dataKey="value"
+          isAnimationActive={false}
+          baseLine={200}
+          points={data}
+          label={({ index, x, y }: { index: number; x: number; y: number }) => <Label index={index} x={x} y={y} />}
+        />
+      </Surface>,
+    );
+
+    expect(container.querySelectorAll('.customized-label')).toHaveLength(data.length);
+  });
+
+  test('Render customized dot when dot is set to be a function', () => {
+    const renderDot = (props: { cx: number; cy: number }) => {
+      const { cx, cy } = props;
+
+      return <circle key={cx} x={cx} y={cy} r={5} className="customized-dot" />;
+    };
+    const { container } = render(
+      <Surface width={500} height={500}>
+        <Area dataKey="value" isAnimationActive={false} baseLine={200} points={data} dot={renderDot} />
+      </Surface>,
+    );
+
+    expect(container.querySelectorAll('.customized-dot')).toHaveLength(data.length);
```

Comment:
Nit: You could consider to follow best practice guide and query via `screen` as opposed to container. 

https://stackoverflow.com/questions/61482418/react-testing-library-screen-vs-render-queries